lib/plotting_functions.py

```diff
@@ -1218,7 +1218,7 @@ def plot_map_and_save(wks, case_nickname, base_nickname,
 
     # get statistics (from non-wrapped)
     fields = (mdlfld, obsfld, diffld, pctld)
-    area_avg = [global_average(x, wgt) for x in fields]
+    [spatial_average(x, weights=wgt, spatial_dims=None) for x in fields]
```

Comment:
Does this still need to be saved as `area_avg`?

Reply:
Nice catch, this was declared in my branch. I wonder what happened. Anyways, just updated it thanks!